sdk/core/Azure.Core/tests/samples/DynamicJsonSamples.cs

```diff
@@ -229,6 +229,27 @@ public void SetWidgetDynamicJson()
             #endregion
         }
 
+        [Test]
+        public void DisposeDynamicJson()
+        {
+            WidgetsClient client = GetMockClient();
+            dynamic details = null;
+
+            #region Snippet:AzureCoreDisposeDynamicJson
+            Response response = client.GetLargeWidget();
+            using (dynamic widget = response.Content.ToDynamicFromJson(DynamicCaseMapping.PascalToCamel))
+            {
+#if !SNIPPET
+                details = widget.Details;
```

Comment:
What is this for?

Reply:
To be able to validate the object was disposed on line 250 below.